com.github.chipmuenk.pyfda.yaml

```diff
@@ -0,0 +1,373 @@
+app-id: com.github.chipmuenk.pyfda
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+# when the runtime version changes, check if python version changed, because in pyqt5 the
+# python version is hardcoded!
+command: pyfdax
+rename-icon: pyfda_icon #Image will renamed to match the app-id konvention
+rename-appdata-file: pyfda.appdata.xml
+rename-desktop-file: pyfda.desktop
+finish-args:
+- "--share=ipc"
+- "--socket=wayland"
+- "--socket=fallback-x11"
+- "--device=dri"
+- "--own-name=com.github.chipmuenk.pyfda"
+- "--filesystem=home"
```

Comment:
does it really need home filesystem access.

Reply:
Currently I'm using a `~/.pyfda` directory for configuration files that is created during the first startup. For a `pip install pyfda` this shouldn't be a problem, but for flatpak installation I'm not happy with the broad access rights to do this either. I'm also not sure what "home filesystem access" means when you install the flatpak system-wide (i.e. with elevated rights). 

Do you have a suggestion how to circumvent this problem? I do need some read / write access right for importing / exporting filter designs etc.

Reply:
for the configuration file we might use one of the xdg dirs:
https://docs.flatpak.org/en/latest/flatpak-command-reference.html
maybe FLATPAK_CONFIG_DIR?

Reply:
`--persist=.pyfda` would persist the directory. until the app support xdg.

Reply:
I set persist

Reply:
If you're passing `--filesystem=home`, `--persist` has no effect.